### DISCUSSION TITLE
My control is  Dead! Help me ! PLSSSSS!

### DISCUSSION THREAD

**Discussion by djgel** [Yogui79/IntexPureSpa#70] [General]
Hello friends!

Need help! unfortunately my control stopped working, I opened it and changed the battery and nothing. I saw that at the base there are some components getting VERY hot! The project needs to mediate the communication between the two, it alone is not capable of establishing communication alone and being independent of control. In turn, the control is no longer on the market, the only one I found of my version (before 2021) is in Poland and they don't ship, besides being expensive!

But anyway, is there any way to send the commands directly to the card's processor, such as using the TX and RX for this?

Any Ideas? Can't the J3 be used for something?

Thanks!

**Comment by djgel** [comment]
From what I read here, several people tried and failed to find a way to connect directly to the main board, forgetting the control altogether.


In this case, I am developing an MAIN BOARD based ESP32 to replace the Original Main Board!

I just need the voltage and amperage of the Hard Water Coil (black wire with blue plug, G & H identification here), can anyone help get this ??

Finished the circuits in Kicad and codes being developed!

Initially activation and use only through MQTT, but I intend to use a WEB in the second stage.


![image](https://github.com/Yogui79/IntexPureSpa/assets/1890658/80b11053-68ef-4cb4-83ae-6816db7b0e9c)

**Reply by BenoitB31** [reply]
hello, have you got an answer? because I broke my card and I can mesure for you and after I am interesting by your project.
